prow/istio-presubmit.sh

```diff
@@ -64,6 +64,27 @@ else
 fi
 cd $ROOT
 
+# go test setup
+time dep ensure -v
+#mkdir -p ~/.kube
+#cp ./.circleci/config ~/.kube/config
+#ln -sf ~/.kube/config ./pilot/platform/kube/config
+mkdir -p ~/envoy
+cd ~/envoy
+ISTIO_PROXY_BUCKET=$(sed 's/ = /=/' <<< $( awk '/ISTIO_PROXY_BUCKET =/' $ROOT/WORKSPACE))
+PROXYVERSION=$(sed 's/[^"]*"\([^"]*\)".*/\1/' <<<  $ISTIO_PROXY_BUCKET)
+PROXY=debug-$PROXYVERSION
+wget -qO- https://storage.googleapis.com/istio-build/proxy/envoy-$PROXY.tar.gz | tar xvz
+ln -sf ~/envoy/usr/local/bin/envoy $ROOT/pilot/proxy/envoy/envoy
+cd $ROOT
+
+# go test execution
+echo FIXME remove mixer tools exclusion after tests can be run without bazel
```

Comment:
Why not go test ./...

Reply:
Good question. For some reason when I run `./...` it gives a lot of errors related to path not found... any idea why ?
```
guptasu-macbookpro:istio guptasu$ go build ./...
vendor/cloud.google.com/go/cmd/go-cloud-debug-agent/internal/breakpoints/breakpoints.go:23:2: cannot find package "golang.org/x/debug" in any of:
	/Users/guptasu/go/src/istio.io/istio/vendor/golang.org/x/debug (vendor tree)
	/usr/local/go/src/golang.org/x/debug (from $GOROOT)
	/Users/guptasu/go/src/golang.org/x/debug (from $GOPATH)
vendor/cloud.google.com/go/internal/testutil/cmp.go:25:2: cannot find package "github.com/google/go-cmp/cmp" in any of:
	/Users/guptasu/go/src/istio.io/istio/vendor/github.com/google/go-cmp/cmp (vendor tree)
	/usr/local/go/src/github.com/google/go-cmp/cmp (from $GOROOT)
	/Users/guptasu/go/src/github.com/google/go-cmp/cmp (from $GOPATH)
vendor/cloud.google.com/go/profiler/profiler.go:54:2: cannot find package "github.com/google/pprof/profile" in any of:
	/Users/guptasu/go/src/istio.io/istio/vendor/github.com/google/pprof/profile (vendor tree)
	/usr/local/go/src/github.com/google/pprof/profile (from $GOROOT)
	/Users/guptasu/go/src/github.com/google/pprof/profile (from $GOPATH)
vendor/github.com/Azure/go-autorest/autorest/azure/auth/authfile.go:31:2: cannot find package "github.com/dimchansky/utfbom" in any of:
	/Users/guptasu/go/src/istio.io/istio/vendor/github.com/dimchansky/utfbom (vendor tree)
	/usr/local/go/src/github.com/dimchansky/utfbom (from $GOROOT)
	/Users/guptasu/go/src/github.com/dimchansky/utfbom (from $GOPATH)
vendor/github.com/apache/thrift/lib/go/test/tests/protocol_mock.go:26:2: cannot find package "thrift" in any of:
	/Users/guptasu/go/src/istio.io/istio/vendor/thrift (vendor tree)
	/usr/local/go/src/thrift (from $GOROOT)
	/Users/guptasu/go/src/thrift (from $GOPATH)
vendor/github.com/apache/thrift/lib/go/test/tests/thrifttest_driver.go:25:2: cannot find package "thrifttest" in any of:
	/Users/guptasu/go/src/istio.io/istio/vendor/thrifttest (vendor tree)
	/usr/local/go/src/thrifttest (from $GOROOT)
	/Users/guptasu/go/src/thrifttest (from $GOPATH)
vendor/github.com/apache/thrift/test/go/src/bin/stress/main.go:25:2: cannot find package "gen/stress" in any of:

```